### PR TITLE
[WIP] massive rewrite (help and comments welcome)

### PR DESCRIPTION
**Pull request by devsnek**
woo

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,105 +1,36 @@
...
-    clientId: Config.uservoice.key.trim(),
-    subdomain: Config.uservoice.UVDomain.trim()
-  })
-}
-
-bot.Dispatcher.on(Events.MESSAGE_CREATE, (c) => {
-  if (c.message.channel.id !== Config.discord.feedChannel && c.message.content.match(UVRegex) !== null && c.message.content.indexOf(Config.discord.prefix) !== 0) {
-    let parts = c.message.content.match(UVRegex)
-    Commands['chatVoteInit'].fn(c.message, parts[2], uvClient)
+const Discord = require('discord.js');
```
Comment (Dougley) [inline index.js:1]: no

File: package.json [hunk 2]
```diff
@@ -15,6 +14,9 @@
   "maintainers": [
     {
       "name": "Benjamin Hoffmeyer (TheDoctorsLife)"
+    },
+    {
+      "name": "snek <me@gus.host>"
```
Comment (Dougley) [inline package.json:19]: no

File: package.json [hunk 3]
```diff
@@ -30,10 +32,8 @@
     "eslint-plugin-standard": "^2.0.1"
   },
   "dependencies": {
-    "bugsnag": "^1.9.1",
-    "discordie": "^0.10.0",
-    "html-entities": "^1.2.1",
-    "require-directory": "^2.1.1",
+    "bugsnag": "^1.11.0",
+    "discord.js": "^11.1.0",
```
Comment (Dougley) [inline package.json:36]: no

File: util/MessageAsk.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const Discord = require('discord.js');
```
Comment (Dougley) [inline util/MessageAsk.js:1]: no

File: util/UserVoice.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+const config = require('../config');
+const UserVoice = require('uservoice-nodejs');
+const snekfetch = require('snekfetch');
```
Comment (Dougley) [inline util/UserVoice.js:3]: no, use superagent.

File: .gitignore [hunk 6]
```diff
@@ -38,17 +38,12 @@ jspm_packages
...
 
 # WebStorm stuff
 .idea
 
-# Channel config
-channels.js
-
-dump.json
\ No newline at end of file
+old
```
Comment (Dougley) [inline .gitignore:49]: what

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by Arzte** [comment]
@devsnek Please fix the outlined problems and update to the current revision of the master branch, as there is several files in conflict.

**Comment by nullpixel** [comment]
Will reopen if you begin work

**Pull request closed without merging**